### DISCUSSION TITLE
這學期有什麼收穫？平常會怎麼提升自己？

### DISCUSSION THREAD

**Discussion by Elaine2899** [1101-GS4719-C/Python-Community-GS4719-C#1503] [General]
又過了一學期，大家這學期有什麼收穫嗎？
以我自己為例，這學期嘗試了一些之前從來沒有接觸過的事物，像是撰寫網頁、拍攝微電影以及接家教學生，這些對我來說都是陌生且困難的。其中是家教教學，怎麼組織及規劃課程、怎麼表達才能讓吸收較慢的學生能夠理解新的觀念？
雖然在學期間摸索到一些新的能力與想法，但究竟該怎麼持續進步以及開始新的自我提升呢？
比如寒假期間很希望自己可以精進英文能力，但過了一個寒假，又過了一個學期，我什麼也沒開始，也不知道該從哪裡開始。
又比如希望可以持續精進寫網頁的相關能力，但又不知道怎麼開始
想問問大家平時是如何提升自己的？不論是語言、溝通、健身或是尋找新知識。

**Comment by xiang305** [comment]
這學期我加入了學校沿海雙語導覽的USR團隊，了解了桃園在地的文化，也看到新舊文化所產生的衝突。更讓我從中反思，我能否用自己的所學為這個社會貢獻什麼。另外，我也參加了通訊營的籌備、圍棋棋院師培課程等等...。我主要是透過參加各類的活動來充實自己，拓展眼界，讓自己對當今的社會有更進一步的了解
